docs/DevDocs/IngestingData.md

```diff
@@ -1,26 +1,33 @@
 ### Ingesting Data
 
 ## RDWATCH_MODEL_RUN_API_KEY
-Check the `./dev/.env.docker-compose` environment file or the `.env` file in the root of the repository for the RDWATCH_MODEL_RUN_API_KEY.
-This key is a special key used for services outside of the standard Django login to be able to push data into the system.
-The key will be used in the scripts and in the headers for pushing data into the system.
+Check the `./dev/.env.docker-compose` environment file or the `.env` file in the root of the repository for the presence of the `RDWATCH_MODEL_RUN_API_KEY` variable.
+This is a special key used for services outside of the standard Django login to be able to push data into the system.
+The key will be used in the scripts and in the headers for pushing data into the system. Copy the value from that file when running the below script against a local instance. When running against a production deployment, you'll need to acquire an API key for that instance and use that instead.
 
 
 ## Loading Ground Truth Data
-Within the ./scripts directory is a python script named `loadGroundTruth.py`.  This file can be used in conjunction with the ground truth annotaitons located in the annotation Repo:
+Within the `scripts` directory is a python script named `loadGroundTruth.py`.  This file can be used in conjunction with the ground truth annotations located in the annotation Repo:
 [Annotation Repo](https://smartgitlab.com/TE/annotations)
-Running a command like `python loadGroundTruth.py ~/AnnotationRepoLocation --rgd-api-key {RDWATCH_MODEL_RUN_API_KEY}` will load all of the annotations for the ground truth while skipping the regions.
+Running a command like:
+
+```bash
+python loadGroundTruth.py ~/AnnotationRepoLocation --rgd-api-key {RDWATCH_MODEL_RUN_API_KEY}
+```
+
+will load all of the annotations for the ground truth while skipping the regions.
```

Comment:
```suggestion
will load all of the annotations for the ground truth along with the regions.  using `--skip_regions` will skip loading the region geometry
```

My mistake, I think I modified the command from the source `.md` file and forgot to update the associated text.